ethsigner/core/src/test/java/tech/pegasys/ethsigner/core/jsonrpcproxy/EthAccountsBodyProviderTest.java

```diff
@@ -93,4 +95,24 @@ public void missingParametersIsOk() {
     assertThat(jsonObj.getInteger("id")).isEqualTo(id);
     assertThat(jsonObj.getJsonArray("result")).containsExactly(address);
   }
+
+  @Test
+  public void multipleValueFromBodyProviderInsertedToResult() {
```

Comment:
Can a test be added that proves the correct set of accounts is returned, if the accounts are changed after the EthAccountsBodyProvider has been created. So that the case you fixed before in the where it wasn't dynamic is tested.